Match desktop Chrome for evergreen Android WebView versions

antross opened the pull request:
Fix #374 

I did some digging into the MDN [browser-compat-data](https://github.com/mdn/browser-compat-data) for a second opinion on version history for the Android WebView. Based on this it appears a more complete way to account for missing version numbers in the `caniuse` data (short of using the MDN data directly) would be to match the version numbers after the Android WebView went evergreen (version 37) with desktop Chrome. 

https://github.com/mdn/browser-compat-data/blob/master/browsers/webview_android.json
https://github.com/mdn/browser-compat-data/blob/master/browsers/chrome.json

This gets `last N versions` and related queries working correctly and resolves another issue with `caniuse` data currently being about seven versions behind the current version of the Android WebView.

This reliable of a relationship between versions doesn't necessarily apply to all mobile browsers, so I left the recently added `mobileToDesktop` mapping untouched.

antross commented:
Looking into the test failures. Thought I had run this locally, but apparently not with my changes... 😞

ai commented:
1. Check that you commit all changes( maybe you forget to commit a file)
2. Try to run `yarn test` instead of `npx jest`, etc

I am preparing to conference talk right now. I will investigate this problem a few days later.

antross commented:
Turns out I just neglected to re-run the tests after my last edit which also matched `and_chr` 🤦‍♂️

I fixed it by removing that edit and re-scoping this PR to just match data for the Android WebView since that was the original focus. I also added a test case to validate the version transition from "before evergreen" (up to `android 4.4.3-4.4.4`) and "after evergreen" (`android 37` and beyond).

ai commented:
Oops. I missed one big problem https://github.com/browserslist/browserslist/issues/374#issuecomment-495583421. I reverted PR manually.
